cmd/create/cluster/cmd.go

```diff
@@ -996,6 +996,67 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+	if err != nil {
+		r.Reporter.Errorf("%s", err)
+		os.Exit(1)
+	}
+
+	billingAccounts := ocm.GenerateBillingAccountsList(cloudAccounts)
+
+	if billingAccount == "" && !interactive.Enabled() {
+		// if a billing account is not provided we will try to use the infrastructure account as default
+		if helper.ContainsPrefix(billingAccounts, awsCreator.AccountID) {
+			billingAccount = awsCreator.AccountID
+			r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+			r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+		} else {
+			r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters.")
+		}
+	}
+
+	if interactive.Enabled() {
+		if len(billingAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  billingAccounts,
+			})
+
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid billing account: '%s'", err)
+				os.Exit(1)
+			}
+
+			billingAccount = aws.ParseOption(billingAccount)
+		}
+
+		if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
+			r.Reporter.Errorf("Expected a valid billing account")
+			os.Exit(1)
+		}
+
+		// Get contract info
+		contracts, isContractEnabled := GetBillingAccountContracts(cloudAccounts, billingAccount)
+
+		if billingAccount != awsCreator.AccountID {
+			r.Reporter.Infof("The selected AWS billing account is a different account than your AWS infrastructure account." +
+				"The AWS billing account will be charged for subscription usage. " +
+				"The AWS infrastructure account will be used for managing the cluster.")
+		} else {
+			r.Reporter.Infof("Using '%s' as billing account.",
+				billingAccount)
+		}
+
+		if isContractEnabled && len(contracts) > 0 {
+			//currently, an AWS account will have only one ROSA HCP active contract at a time
+			contractDisplay := GenerateContractDisplay(contracts[0])
```

Comment:
Is this necessary? Do we have to display this information? 

Also the fact that it currently only has one active contract, suggests this could change, requiring new release of ROSA to fix this print.

Reply:
Yeah it's from the design: https://docs.google.com/document/d/10QqWHZLtoK2cGrTWO4-wh-ElT-VCoFbLlqtTcFdjtYs/edit#heading=h.vj53sho39lu0

Regarding to only have one active contract, I believe that is the current design of the CRD: 
![image](https://github.com/openshift/rosa/assets/26282541/e34f6cb0-8db4-436a-967f-3e429f2950ea)

@ciaranRoche You are correct that we might need to change this behaviour, but I believe it's more a feature instead of a fix. As of now, UI also treats the display the same way. With the introduction of multiple contracts, both UI and backend would have to accommodate that. Current design is only supporting one contract. 

However, I can also change this to a for loop if you think it might be better for long-term usage. WDYT?

<img width="958" alt="image" src="https://github.com/openshift/rosa/assets/26282541/c2420954-eb0c-440c-8228-03181d017edd">